stetho/src/main/java/com/facebook/stetho/screencast/android/ScreencastDispatcher.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.screencast.android;
```

Comment:
We need to move this class into a package over in `com.facebook.stetho.inspector` .... something like `com.facebook.stetho.inspector.screencast.ScreencastDispatcher` sounds pretty good